go/kbnm/hostmanifest/whitelist.go

```diff
@@ -2,10 +2,21 @@ package hostmanifest
 
 import (
 	"encoding/json"
+	"fmt"
 	"os"
 	"path/filepath"
 )
 
+func wrapWriteErr(err error, hostsPath string) error {
+	if !os.IsPermission(err) {
+		return err
+	}
+	dirName := filepath.Dir(hostsPath)
+	return fmt.Errorf("%s: Make sure you are the owner of the directory. "+
+		"You can run:\n "+
+		"  sudo chown -R $(whoami):staff %q", err, dirName)
```

Comment:
Er, good point, this used to be only in _darwin.go land and it got moved. Hmmmm.